dashboard/app/models/blockly.rb

```diff
@@ -294,4 +304,12 @@ def autoplay_blocked_by_level?
     # Wrapped since we store our serialized booleans as strings.
     self.never_autoplay_video == 'true'
   end
+
+  # Returns an array of all the contained levels
+  # (based on the contained_level_names property)
+  def contained_levels
+    names = properties["contained_level_names"]
+    return [] unless names.present?
+    Level.where(name: properties["contained_level_names"])
```

Comment:
I remember you brought up the issue of referencing levels by name at runtime.  Can we resolve the level IDs during init, then look up the levels via `Script.cache_find_level`?